src/fsharp/ast.fs

```diff
@@ -135,9 +135,9 @@ type ParserDetail =
 
 // PERFORMANCE: consider making this a struct.
 [<System.Diagnostics.DebuggerDisplay("{idText}")>]
-[<Sealed>]
+[<Struct>]
```

Comment:
I'm only now revieing these changes

Do we know for sure this as a good idea? ``range`` is already a largish struct, and we had previously held off making ``Ident`` a struct too until we had some data